en/docs/learn/configuring-oauth2-openid-connect.md

```diff
@@ -56,7 +56,8 @@ the users with an authorization server-based authentication.
     
         If you want to change the format in which the **Client Secret** is
         stored, open the `	<IS_HOME>/repository/conf/deployment.toml	` file and add the following configuration.
-        ```xml
+
+        ```toml
         [oauth]
 		hash_tokens_and_secrets = true 
```

Comment:
Make sure this is aligned. Doesn't seem proper in the git view.

Reply:
this is fine in the editor